lib/file.js

```diff
@@ -33,6 +33,20 @@ var fs = require('fs');
 //    <!-- build:css /foo/css/site.css -->
 // then dest will equal foo/css/site.css (note missing trailing /)
 //
+var getAttributes = function (attributes) {
+  var tmp = {};
+  return attributes.reduce(function (previousValue, currentValue) {
+    tmp[ currentValue.name ] = currentValue.value;
```

Comment:
What a about `data` attributes or anything else that uses hash as ng-if, my-custom-attribute?